reporter/src/main/kotlin/reporters/NoticeReporter.kt

```diff
@@ -32,13 +32,13 @@ class NoticeReporter : AbstractNoticeReporter() {
         val analyzerResult = ortResult.analyzer!!.result
         val scanRecord = ortResult.scanner!!.results
 
-        val licenseFindings = sortedMapOf<String, SortedSet<String>>()
+        var licenseFindings = sortedMapOf<String, SortedSet<String>>()
```

Comment:
Changing to `var` seems unnecessary.

Reply:
Fixed, thanks, that was a left-over from playing around.